x-pack/test/functional/page_objects/dataset_quality.ts

```diff
@@ -136,7 +140,7 @@ export function DatasetQualityPageObject({ getPageObjects, getService }: FtrProv
       );
     },
 
-    getDatasetsTable(): Promise<WebElementWrapper> {
+    async getDatasetsTable(): Promise<WebElementWrapper> {
```

Comment:
nit: why this change? seems like it doesn't need to be async?

Reply:
Ooopss, I was testing something else, good catch